app/src/main/AndroidManifest.xml

```diff
@@ -1,66 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
The only actual change in the manifest is adding the new activity, the rest is just autoformatting changes